### ISSUE TITLE
Hiding of un-needed fields in the UI

### ISSUE THREAD

**Issue by marko-arn** [SORMAS-Foundation/SORMAS-Project#3866]
### Feature Description

Fields that are not needed to be filled in or where the datas can be prefilled should be hide.

### Problem Description

In the UI are some fields, which are, as far as we discovered, are not used or could be prefilled for Germany.

They are:

![grafik](https://user-images.githubusercontent.com/74791118/102979168-fa371900-4505-11eb-967c-de1200fb1945.png)
In Germany sormas is currently only used for Covid-19, so ist should be pre-filled and the field can hide.

This bith fields are also not used by us:

![grafik](https://user-images.githubusercontent.com/74791118/102979358-3ff3e180-4506-11eb-9e1a-b95fe8574d56.png)

The area-type, ist also not needed:

![grafik](https://user-images.githubusercontent.com/74791118/102979599-92350280-4506-11eb-8677-7feb989bd971.png)

When the new field for guardians are included, the fields to the name of the mother/father and maiden name of the mother are obsolete:
![grafik](https://user-images.githubusercontent.com/74791118/102979694-b7c20c00-4506-11eb-8553-f7a3e3a2945f.png)

### Proposed Change

Hiding the mentioned fields.

### Additional Information

Screenshots are from https://test.sormas.netzlink.com

**Comment by kwa20** [comment]
@Marko-ilmkreis Such decisions are made for a whole country thus far and can therefore not easily be applied when other health departments use the mentioned fields. Instead, customization options will be enhanced in the future.

**Comment by Marko-ilmkreis** [comment]
@kwa20 Thats true, but I thought that maybe the implementation can be done depending on the country of installation.

**Comment by kwa20** [comment]
@Marko-ilmkreis That's correct. The difficulty arises because diverse usage happens within one country.

**Comment by Marko-ilmkreis** [comment]
@kwa20 okay. It would be time to have some forum or similar for the admins of Sormas in Germany to interact together. This would be fine for suggestions and you could also have a wider feedback on ongoing changes.

**Comment by bernardsilenou** [comment]
I do not know if the issue was already created,  @MateStrysewske can confirm this. The idea was to have the "flexible form creator" + the option to "hide of show certain variable". A similar designed as we used in the APMS project.
Since this is a big change, @markusmann-vg  would inform us on when to begin working on this.

**Comment by MateStrysewske** [comment]
@bernardsilenou That's pretty far away.

**Comment by Lucas-Walter-GA-Tuebingen** [comment]
This would be useful for LK - Tübingen as well
